executor/common_linux.h

```diff
@@ -1466,6 +1466,98 @@ static long syz_extract_tcp_res(volatile long a0, volatile long a1, volatile lon
 #include "common_usb_linux.h"
 #endif
 
+#if SYZ_EXECUTOR || __NR_syz_raw_bind_info
+#include <linux/raw.h>
+#include <string.h>
+#include <sys/stat.h>
+#include <sys/sysmacros.h>
+static long syz_raw_bind_info(int fd, int arg)
+{
+	// syz_raw_bind_info$char_raw(dev fd, raw_dev int32[1:2]) block_info_ptr
+	// raw_dev is used to allow for multiple bindings per process.
+	struct stat sb;
+	int result = fstat(fd, &sb);
+	if (result == -1) {
+		return -1;
+	}
+	struct raw_config_request* rq = (struct raw_config_request*)malloc(sizeof(struct raw_config_request));
```

Comment:
See #1916

Reply:
All memory that the fuzzer operates with should be allocated within the "data" region, that is, be fuzzer-allocated.